net/pfSense-pkg-pfBlockerNG/files/usr/local/www/pfblockerng/pfblockerng_feeds.json

```diff
@@ -748,16 +748,16 @@
 			]
 		},
 		"VPN_4": {
-			"info":		"A list of common VPN providers",
-			"description":	"List of common VPN providers.",
+			"info":		"A list of IP's used by VPN providers",
```

Comment:
nit: I guess people use it, but the "correct" plural would be "IPs".
Even better would be "IP addresses"